deploy/custom_resources/cognito_config/cognito_urls_config.py

```diff
@@ -103,11 +103,11 @@ def setup_cognito(
 
     if with_approval_tests == 'True':
         ssm = boto3.client('ssm', region_name=region)
```

Comment:
We need to update the instructions in the README file for users to create the correct SSM parameter

Reply:
done